pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -95,6 +95,9 @@ type PodStats struct {
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	VolumeStats []VolumeStats `json:"volume,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
+	// EphemeralStorage reports the total filesystem usage for the containers and emptyDir volumes in the measured Pod.
```

Comment:
nit: change to "emptyDir-backed" volumes

Reply:
fixed